assimilation_code/modules/io/direct_netcdf_mod.f90

```diff
@@ -1507,23 +1528,23 @@ end subroutine clamp_variable
 !-------------------------------------------------------------------------------
 
 subroutine write_variables(ncid, var_block, start_var, end_var, domain, &
-                           do_file_clamping, force_copy)
+                           do_variable_clamping, force_copy)
 
 integer,  intent(in)    :: ncid
 real(r8), intent(inout) :: var_block(:)
 integer,  intent(in)    :: start_var
 integer,  intent(in)    :: end_var
 integer,  intent(in)    :: domain
-logical,  intent(in)    :: do_file_clamping
+logical,  intent(in)    :: do_variable_clamping
```

Comment:
This is used in a loop around variables, not files. The direct_netcdf_mod.f90 calls this function with a variable called 'clamp_vars' ....